content/notebooks/data_discovery_and_access/data_discovery_and_access.ipynb

```diff
@@ -289,9 +304,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Downloading Files Locally (not recommended)\n",
+    "## Downloading Files (not recommended)\n",
+    "\n",
+    "Though it is **not recommended**, there may be instances where data files must be downloaded for certain specific science cases. To do that, we can use the URIs and the `S3FileSystem.get` function (documentation [here](https://s3fs.readthedocs.io/en/latest/api.html#s3fs.core.S3FileSystem.get)). Running the below cell will download the data to your personal instance of the science platform, however the same code could be combined with some of the above cells and run on your local machine to download the data to a private computer.\n",
```

Comment:
I would add the reason we don't recommend downloading files is due to the number and size of files that we expect will make up Roman survey data. Users are encouraged to adopt a workflow similar to the file streaming described above for the best experience.